test_launch_ros/test/test_launch_ros/actions/test_node.py

```diff
@@ -191,6 +192,13 @@ def test_create_node_with_invalid_parameters(self):
         )
         self._assert_launch_no_errors([node_action])
 
+    def test_deprecated_node_parameters(self):
+        node_action = launch_ros.actions.Node(
+            package='demo_nodes_py', node_executable='talker_qos', output='screen',
+            node_name='my_node', node_namespace='my_ns'
+        )
+        self._assert_launch_no_errors([node_action])
+
```

Comment:
@jacobperron should we also ensure that it does throw if both `name` and `node_name` are provided? Same for namespace.

Reply:
Added tests c19dc08